src/foam/nanos/app/AppConfig.js

```diff
@@ -19,13 +19,39 @@ foam.CLASS({
     },
     {
       class: 'String',
-      name: 'copyright'
+      name: 'privacy',
+      value: 'Privacy Policy'
+    },
+    {
+      class: 'String',
+      name: 'privacyUrl',
+      value: 'https://nanopay.net/privacy-policy'
+    },
+    {
+      class: 'String',
+      name: 'copyright',
+      value: 'Copyright © 2018 Nanopay. All rights reserved.'
     },
     {
       class: 'String',
       name: 'url',
       value: 'http://localhost:8080/'
     },
+    {
+      class: 'String',
+      name: 'urlLabel',
+      value: 'Powered by Nanopay'
```

Comment:
Believe this should be nanopay with lowercase 'n'.

Reply:
This and any other reference to nanopay explicitly should be removed and set elsewhere in our repositories

Reply:
This and any other references to nanopay should be removed from this repository and stored in one of our repos

Reply:
"FOAM Powered" might be a suitable default.